packages/home-service/src/helpers.ts

```diff
@@ -1,3 +1,67 @@
 // write your helper functions here
-import { RedisPubSub } from 'graphql-redis-subscriptions';
-export const pubsub = new RedisPubSub();
+const fetch = require('node-fetch');
+
+class HomeAPIHelper {
+    userFragment = `
+                fragment user on UserType {
+                    _id
+                    name
+                    title
+                    uid
+                    rhatUUID
+                    memberOf
+                    isActive
+                    timestamp {
+                        createdAt
+                        createdBy {
+                                kerberosID
+                                name
+                        }
+                    }
```

Comment:
THis strructure is also changed in last PR

Reply:
Okay, thanks. Changed fragment according to the new UserType